src/System.Windows.Forms/src/System/Windows/Forms/ListViewGroup.cs

```diff
@@ -76,6 +77,19 @@ public ListViewGroup(string? header, HorizontalAlignment headerAlignment) : this
             _headerAlignment = headerAlignment;
         }
 
+        internal AccessibleObject? AccessibilityObject
+        {
+            get
+            {
+                if (_accessibilityObject is null)
+                {
+                    _accessibilityObject = new ListViewGroupAccessibleObject(this, IsDefault);
```

Comment:
```suggestion
                    bool isDefaultGroup = ListView?.Groups.Contains(this) == false;
                    _accessibilityObject = new ListViewGroupAccessibleObject(this, isDefaultGroup);
```

`ListView != null && !ListView.Groups.Contains(this)` --> `ListView != null && ListView.Groups.Contains(this) == false` --> `ListView?.Groups.Contains(this) == false`